makefile

```diff
@@ -43,18 +43,18 @@ run:
 
 build:
 	${CC} build ${RFLAGS}
+	strip --strip-debug $(RBIN)
+
+bench:
+	hyperfine "target/release/rsfetch -NcldkuUH@swp xbps" \
+		kfetch ufetch pfetch \
+		"aura -n \"term\"" \
+		"neofetch --disable resolution --disable theme --disable icons --disable term --disable cpu --disable memory"
```

Comment:
No `screenfetch`? `screenfetch` is arguably the number one competitor to `neofetch`. It'd be good to get it in the benchmarks.

Reply:
good point. I'll add it (and remove kfetch, which was supposed to be temporary :-))